CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -958,9 +961,48 @@ void IRenderWebBrowser.OnCursorChange(IntPtr handle, CursorType type, CursorInfo
             }
         }
 
+        private Visual GetParentWindow()
```

Comment:
Seems like you are reinventing the wheel here, what was the problem using the build in function? 

https://docs.microsoft.com/en-us/dotnet/api/system.windows.window.getwindow?view=netframework-4.8